CMakeLists.txt

```diff
@@ -1,3 +1,57 @@
-cmake_minimum_required(VERSION 3.10.0)
+cmake_minimum_required(VERSION 3.16.0)
 project(bgfx_map VERSION 0.1.0)
-add_executable(bgfx_map main.cpp)
\ No newline at end of file
+set(SOURCE_FILES main.cpp)
+Include(FetchContent)
+FetchContent_Declare(
+    bx
+    GIT_REPOSITORY https://github.com/bkaradzic/bx.git
+    GIT_TAG master
+)
+FetchContent_GetProperties(bx)
+if(NOT bx_POPULATED)
+    FetchContent_Populate(bx)
+    execute_process(COMMAND git submodule update --init --recursive
+    WORKING_DIRECTORY ${bx_SOURCE_DIR})
+    execute_process(COMMAND ${CMAKE_COMMAND} -E create_symlink ${bx_SOURCE_DIR} ${CMAKE_BINARY_DIR}/_deps/bx)
+    execute_process(COMMAND make ${PLATFORM} -C ${CMAKE_BINARY_DIR}/_deps/bx/)
```

Comment:
Move compilation of library from makefile generation phase to compilation phase

Reply:
@pan-diaxik thank you, issue fixed 👍